pkg/network/ebpf/c/protocols/protocol-classification-helpers.h

```diff
@@ -30,7 +69,7 @@ static __always_inline bool is_http2(const char* buf, __u32 buf_size) {
 #define HTTP2_SIGNATURE "PRI * HTTP/2.0\r\n\r\nSM\r\n\r\n"
 
     bool match = !bpf_memcmp(buf, HTTP2_SIGNATURE, sizeof(HTTP2_SIGNATURE)-1);
-    
+
```

Comment:
Why the extra space?

Reply:
there is not extra space
my IDE removes any trailing spaces that were left in the code